packages/react/src/__stories__/Typography.stories.tsx

```diff
@@ -1,18 +1,128 @@
+        har for eksempel klassen <code>paragraph</code>.
+      </p>
+      <h3 className="heading-m">Sitater</h3>
+      <p className="paragraph">
+        Lengre sitater kan framheves med klassen <code>blockquote</code>:
+      </p>
+      <blockquote className="blockquote">
+        Det er et tøft boligmarked, og med de prisene det er på bolig i Oslo,
+        virket det nesten umulig å komme inn på markedet. Jeg var på noen
+        visninger, men det ble enten for lite eller for gammelt og slitt
+      </blockquote>
+      <h3 className="heading-m">Bildetekster</h3>
+      <p className="paragraph">
+        Klassen <code>description</code> kan f.eks. brukes for bildetekster:
+      </p>
+      <figure>
+        <img
+          className="mb-4 max-w-96 bg-blue-dark p-4"
+          src="https://res.cloudinary.com/obosit-prd-ch-clry/image/upload/q_auto/v1619689575/OBOS%20Merkevare/OBOS/Liggende/obos_liggende_hus_hvit.svg"
+          alt="OBOS logo"
+        />
+        <figcaption className="description">
+          OBOS sin logo har hvit tekst, og bildet må derfor ha en mørk bakgrunn.
+          Slik at man kan se hva det står.
+        </figcaption>
+      </figure>
+    </div>
+  );
 };
 
-export default meta;
+export const Headings = () => {
+  return (
+    <>
+      <h1 className="heading-xl">Overskrift nivå 1</h1>
+      <h2 className="heading-l">Overskrift nivå 2</h2>
+      <h3 className="heading-m">Overskrift nivå 3</h3>
+      <h4 className="heading-s">Overskrift nivå 4</h4>
+    </>
+  );
+};
+
+export const Lead = () => (
+  <p className="lead">
+    Dokumentavgift er en avgift som du må betale til staten når du kjøper en
+    fast eiendom.
+  </p>
+);
 
-export const Icons = () => {
+export const Body = () => {
   return (
-    <div className="flex flex-col gap-4">
-      <h1 className="h1">This is a H1 heading</h1>
-      <h2 className="h2">This is a H2 heading</h2>
-      <h3 className="h3">This is a H3 heading</h3>
-      <h4 className="h4">This is a H4 heading</h4>
-    </div>
+    <>
+      <p className="paragraph">
+        Mange opplever at prisvekst og rentehevinger har fått store konsekvenser
+        for økonomien. Førstegangskjøpere og de som tjener mindre, sliter med å
+        komme seg inn på boligmarkedet.
+      </p>
+      <p className="paragraph">
+        – Det er et tøft boligmarked, og med de prisene det er på bolig i Oslo,
+        virket det nesten umulig å komme inn på markedet. Jeg var på noen
+        visninger, men det ble enten for lite eller for gammelt og slitt,
+        forteller boligkjøper Brita (30).
+      </p>
+    </>
   );
 };
+
+export const Blockquote = () => (
+  <blockquote className="blockquote">
+    Det er et tøft boligmarked, og med de prisene det er på bolig i Oslo, virket
+    det nesten umulig å komme inn på markedet. Jeg var på noen visninger, men
+    det ble enten for lite eller for gammelt og slitt
+  </blockquote>
```

Comment:
Kan vi finne et litt bedre sitat siden dette er "public"? Det føltes lite flatterende for OBOS ute av kontekst :D

Reply:
Hehe, ja det er jeg enig i!